### PR TITLE
changelog: start of changelog for 0.3.0

### PR DESCRIPTION
**Pull request by ehazlett**
This is the start of the changelog for 0.3.0.

**Not to be merged before release**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,66 @@
...
+- New Provisioning system to allow for greater flexibility and stability for installing and configuring Docker
+- RedHat provisioner (use RHEL as base OS) - (experimental)                     
+- PowerShell support (configure Windows Docker CLI)                             
+- Command Prompt (cmd.exe) support (configure Windows Docker CLI)               
+- Filter command help by driver                                                 
+- Ability to import Boot2Docker instances                                       
+- Boot2Docker CLI migration guide (experimental)                                
+- Format option for `inspect` command                                           
+- New logging output format to improve readability and display across platforms 
+- Updated "active" machine concept - now is implicit according to `DOCKER_HOST` environmet variable.  Note: this removes the implicit "active" machine and can no longer be specified with the `active` command.  You change the "active" host by using the `env` command instead.
```
Comment (nathanleclaire) [inline CHANGES.md]: "environment" spelling

Comment (ehazlett) [inline CHANGES.md]: doh thanks!

**Comment by nathanleclaire** [comment]
I'd include a big "BREAKING CHANGES" section with, of course, the fact that we are deprecating the `docker-machine ip`, `docker-machine start`, `docker-machine stop` etc. commands.

**Comment by ehazlett** [comment]
Merging as this is in the current RCs.

**Pull request merged**